tools/site_and_regional/run_neon.py

```diff
@@ -620,11 +626,11 @@ def set_ref_case(self, case):
         case.set_value("RUN_REFDATE", refdate)
         if case_root.endswith(".postad"):
             case.set_value("RUN_STARTDATE", refdate)
-        else:
-            case.set_value(
-                "RUN_STARTDATE",
-                "{yr:04d}-{mo:02d}-01".format(yr=self.start_year, mo=self.start_month),
-            )
+        #else:
```

Comment:
@wwieder I think we should just remove this commented out code. git keeps these changes in the repository, so we can get it back if needed.  Is there a reason to keep it? I'm assuming it doesn't work right if it's used.

Reply:
Yes, this is fine to remove.

Reply:
Done